app/Services/VoucherEvaluator/Evaluations/ScottishChildCanDefer.php

```diff
@@ -30,7 +30,7 @@ public function test($candidate)
         $monthNow = Carbon::now()->month;
         $schoolStartMonth = config('arc.scottish_school_month');
         // Check we're in the start month or the one before.
-        if ($schoolStartMonth - $monthNow > 1) {
+        if (($schoolStartMonth - $monthNow > 1) || ($schoolStartMonth - $monthNow < 0)) {
```

Comment:
resolves to success when:
- a month before (1)
- the month (0)

and fail when _after_ the date OR before 1-month-before

👍 that's my understanding

Reply:
I believe so.